astrophot/models/parameter_object.py

```diff
@@ -17,7 +17,7 @@
 
 class Parameter(object):
     """Object for storing model parameters that are to be optimized in the
-    fitting procedure. A value and it's uncertainty are stored as well
+    fitting procedure. A value and its uncertainty are stored as well
```

Comment:
the `parameter_object.py` file is removed in the new parameter system. I think you'll need to merge the current main branch to remove the appropriate files